Лабораторная работа 5/Task3.py

```diff
@@ -0,0 +1,68 @@
+class Book:
+    """ Базовый класс книги. """
+
+    def __init__(self, name: str, author: str):
+        self._name = name
+        self._author = author
+
+    @property
+    def name(self)-> str:
+        return self._name
+        
+    @property
+    def author(self)-> str:
+        return self._author
+
+    def __str__(self):
+        return f"Книга {self._name}. Автор {self._author}"
+
+    def __repr__(self):
+        return f"{self.__class__.__name__}(name={self._name!r}, author={self._author!r})"
+
+
+class PaperBook(Book):
+    def __init__(self, name: str, author: str, pages: int):
+        super().__init__(name, author)
+        self.pages = pages
```

Comment:
```suggestion
        self._pages = pages
```
Вводим переменную `_pages` чтобы ее защитить и присваивать только после проверки свойств и через setter